src/MARIGOLD/iate.py

```diff
@@ -1,26 +1,51 @@
 from .config import *
 
-def iate(cond, query, z_step = 0.01,
-         dpdz_method = 'LM', void_method = 'driftflux', mueff_method = 'ishii', cd_method = 'doe',  # Method arguments
-         LM_C = 40, k_m = 0.40, LoverD_restriction = 9999,                                          # Pressure drop calculation arguments
-         cheat = True, elbow = False, quarantine = True):                                           # Temporary arguments, fix later
-    """ Calculate the area-averaged interfacial area concentration at query location based on the 1G IATE
+def iate_1d_1g(
+        # Basic inputs
+        cond, query, z_step,
+        
+        # IATE Coefficients
+        C_WE = None, C_RC = None, C_TI = None, alpha_max = 0.75, C = 3, We_cr = 6, acrit_flag = 0, acrit = 0.13, C0 = 1.20,
+
+        # Method arguments
+        dpdz_method = 'LM', cd_method = 'doe', void_method = 'driftflux',
+
+        # Pressure drop calculation arguments
+        LM_C = 40, k_m = 0.40, LoverD_restriction = None,
+
+        # Temporary arguments
+        cheat = True, elbow = False):
+    """ Calculate the area-averaged interfacial area concentration at query location based on the 1D 1G IATE
 
     Version History:
      - v1: Pressure cheating, jgref substitute for jgatm
      - v2: MG update, pressure retrieval, jgatm retrieval
      - v3: Yadav methods, incorporation of COV terms, support for elbows, VU, VD, horizontal
     
     Inputs:
-     - cond:             Condition object, part of MARIGOLD framework
-     - query:            L/D endpoint
-     - z_step:           Axial mesh cell size [-]
-     - void_method:      Void fraction prediction method, 'driftflux' or 'continuity'
+     - cond:            Condition object, part of MARIGOLD framework
+     - query:           L/D endpoint
+     - z_step:          Axial mesh cell size [-]
+
+     - C_WE:            Wake entrainment coefficient
+     - C_RC:            Random collision coefficient
+     - C_TI:            Turbulent impact coefficient
+     - alpha_max:       Maximum void fraction, used for random collision calculation
+     - C:               C
+     - We_cr:           Weber number criterion, used for turbulent impact calculation
+     - acrit_flag:      Enable/disable shutting off turbulence-based mechanisms beyond a critical void fraction
+     - acrit:           Critical void fraction for shutting off turbulence-based mechanisms
+     - C0:              Drift flux coefficient
+
+     - dpdz_method:     Pressure drop prediction method, 'LM' or 'Kim'
+     - cd_method:       Drag coefficient prediction method, 'iter' or 'doe'
+     - void_method:     Void fraction prediction method, 'driftflux' or 'continuity'
 
```

Comment:
For consistency with the comment blocks in Condition, we should add what this function returns. Also not sure what the version history is supposed to do, Git stores all our changes